pkg/controller/controller_binding.go

```diff
@@ -171,7 +171,8 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 
 	serviceClass, servicePlan, brokerName, brokerClient, err := c.getServiceClassPlanAndServiceBrokerForServiceInstanceCredential(instance, binding)
 	if err != nil {
-		return err
+		glog.Warning("couldn't get the things")
```

Comment:
ok this warning isn't very useful.  Can we add more info to it?

Reply:
I can put it back to not having any log message at all.

Reply:
I like the idea of the msg, but I want it to be meaningful